src/poetry/console/commands/init.py

```diff
@@ -303,7 +303,9 @@ def _determine_requirements(
                 ):
                     self.line(f"Adding <info>{package}</info>")
                     result.append(constraint)
-                    package = self.ask("\nAdd a package (leave blank to skip):")
+
```

Comment:
Do I understand correctly that the current code uses different text for the second and subsequent packages, and you have changed it so that it uses the same text for all?

I wonder if that might be a little confusing as a UI: it could feel as though the first package you add "didn't work".  Anyway it seems like a behaviour change that's unrelated to what you're wanting to fix

Reply:
Yeah, I reused the created question that is set the validation. I did not find the text behavior a change to be confusing, but it's not difficult to add an ask with the validate set using the text in the diff above.

Reply:
@dimbleby could you give follow up feedback?

Reply:
I don't really have anything to add to what I said before

- IMO the new behaviour is a little more confusing, I could imagine that for a new user the experience of being asked to add a package, providing a name, and then being asked the exact same question might feel as though it just hadn't worked
- regardless of whether it really is more confusing or not (reasonable people might disagree on such things, and I don't feel especially strongly about it anyway): it's a behaviour change that is unrelated to the fix that you're making

So I'd prefer to see that change undone.

However I'm not going to lose any sleep over it either way.